db/migrate/20151112110911_add_event_id_to_event_logs.rb

```diff
@@ -0,0 +1,6 @@
+class AddEventIdToEventLogs < ActiveRecord::Migration
+  def change
+    add_column :event_logs, :event_id, :integer
+    change_column :event_logs, :event, :string, default: ''
```

Comment:
In production, this table has over 7 million rows. I'm a bit nervous about this migration (and related ones in other PRs) potentially locking the table and blocking all user actions that write to it. Perhaps I'm misremembering the effects of `ALTER TABLE` statements though?

If that is the case, we'll want to schedule these deploys carefully for times of lower usage.